torch/autograd/gradcheck.py

```diff
@@ -254,17 +243,19 @@ def gradgradcheck(func, inputs, grad_outputs=None, eps=1e-6, atol=1e-5, rtol=1e-
         # If grad_outputs is not specified, create random Tensors of the same
         # shape, type, and device as the outputs
         def randn_like(x):
-            var = torch.testing.randn_like(x if x.is_floating_point() else x.double())
+            y = torch.testing.randn_like(x if x.is_floating_point() else x.double(), requires_grad=True)
             if gen_non_contig_grad_outputs:
-                var = torch.testing.make_non_contiguous(var)
-            var.requires_grad = True
-            return var
+                y = torch.testing.make_non_contiguous(y).requires_grad_()
+            return y
```

Comment:
nit: you could remove `requires_grad` from all parts before this line and just do `return y.requires_grad_()`